tutorials/blitz/7_captum_explainability.py

```diff
@@ -0,0 +1,203 @@
+"""
+Explaining Graph Neural Network with Captum
+===========================================
+
+By the end of this tutorial, you will be able to:
+
+- Understand the basic concepts of GNN explainability
+- Use Captum to explain the node classification task
+- Find the most important nodes and edges to the model
+- Visualize the node-centered subgraphs
+
+(Time estimate: 20 minutes)
+
+"""
+
+######################################################################
+# Overviews of GNN Explainability
+# -------------------------------
+# In recent years, research on the interpretability of deep learning models has
+# made significant progress. Compared to cv and nlp domains, there is less
+# research and application of graph model interpretability, yet it is the key
+# to understanding deep graph neural networks. Generally, GNN explanation
+# research often starts with the following tasks: ** Which input edges are more
+# important? Which input nodes are more important? **
```

Comment:
Note that interpretability and explainability are close but not exactly the same. The former one might be associated with white box models like linear regression and decision tree. I will suggest changing this paragraph to

```
Like other deep learning models, GNNs are black box models. This might hinder the deployment of GNNs in risk-sensitive scenarios. Explaining GNNs has been an important research problem. Many GNN explainability methods approach this problem by identifying important nodes and edges in the input graphs.
```